pystoned/CNLS.py

```diff
@@ -223,46 +239,40 @@ def afriat_rule(model, i, h):
 
                 return afriat_rule
 
-        # TODO(error handling): replace with undefined model attribute
-        return False
+        raise ValueError("Undefined model parameters.")
 
     def display_status(self):
         """Display the status of problem"""
         if self.optimization_status == 0:
-            print("Model isn't optimized. Use optimize() method to estimate the model.")
-            return False
+            raise Exception("Model isn't optimized. Use optimize() method to estimate the model.")
         print(self.display_status)
 
     def display_alpha(self):
         """Display alpha value"""
         if self.optimization_status == 0:
-            print("Model isn't optimized. Use optimize() method to estimate the model.")
-            return False
+            raise Exception("Model isn't optimized. Use optimize() method to estimate the model.")
+        if self.rts == RTS_CRS:
+            raise Exception("Estimated intercept (alpha) cannot be retrieved due the CRS model is built without intercept.")

```

Comment:
could be:

`raise Exception("Estimated intercept (alpha) cannot be retrieved due to the constant returns-to-scale assumption.")`

Reply:
Thanks for this precise revision!.

Reply:
The message has been updated